ui/src/containers/Personas/PersonaManage/MergeForm/index.js

```diff
@@ -34,7 +34,7 @@ const renderTarget = ({ hasTarget, mergeTargetId }) => {
   );
 };
 
-const render = ({ id, hasMetadata, getMetadata, setMetadata }) => {
+const MergeForm = ({ id, hasMetadata, getMetadata, setMetadata }) => {
```

Comment:
Naming improves the searchability in our IDE.

Reply:
Yeah good idea 👍